htdocs/xoops_lib/Xoops/Core/Assets.php

```diff
@@ -96,7 +96,7 @@ class Assets
      * @var string string to identify Assetic filters using instanceof
      */
 
-    private $filterInterface = '\Assetic\Filter\FilterInterface';
+    private $filterInterface = '\\Assetic\\Filter\\FilterInterface';
```

Comment:
That is very badly worded. The examples clarify it:
```
// Outputs: You deleted C:\*.*?
echo 'You deleted C:\\*.*?';

// Outputs: You deleted C:\*.*?
echo 'You deleted C:\*.*?';
```
You can use two backslashes, but one will work in all cases *except* when the next character is a single quote or a backslash. I.e. to output `\'` you would have to specify `\\\'`